kratos/integration/integration_info.cpp

```diff
@@ -110,6 +110,10 @@ namespace Kratos
             mNumberOfIntegrationPointsPerSpanVector[DimensionIndex] = 5;
             mQuadratureMethodVector[DimensionIndex] = QuadratureMethod::EXTENDED_GAUSS;
             break;
+        case IntegrationMethod::GI_LOBATTO_1:
+            mNumberOfIntegrationPointsPerSpanVector[DimensionIndex] = 2;
```

Comment:
Shouldn't the value be 1?

Reply:
Maybe I am wrong but, as far as I understand, the `GI_LOBATTO_1` means order 1, right?. Sadly the Lobatto needs 2 points to integrate linear polynomials...

Reply:
Sure. Sorry for the noise.